### PR TITLE
ADD overlay to iab popup

### PR DESCRIPTION
**Pull request by rodrigodaoud**

### CODE REVIEW

File: reference/src/components/popup/popup.less [hunk 1]
```diff
@@ -6,7 +6,7 @@
     bottom: 0;
     left: 0;
     right: 0;
-    z-index: 99999;
+    z-index: 99;
```
Comment (srjsrj) [inline reference/src/components/popup/popup.less]: how did you calculate this value?

File: reference/src/components/popup/popup.less [hunk 2]
```diff
@@ -6,12 +6,11 @@
     bottom: 0;
     left: 0;
     right: 0;
-    z-index: 99;
+    z-index: 1;
```
Comment (srjsrj) [inline reference/src/components/popup/popup.less]: is it tested? I think it could be that ads/other things could overlap message

Comment (rodrigodaoud) [reply reference/src/components/popup/popup.less]: it is, apparently it's not overlaping. all good

**Comment by rodrigodaoud** [comment]
1.5.4-POPO3269

**Pull request closed without merging**